src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { SortType } from '../../types/SortType';
+
+type Props = {
+  activeTodosLength: number;
+  sortType: SortType;
+  setSortType: (sortBy: SortType) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  setSortType,
+  sortType,
+  activeTodosLength,
+}) => {
+  const handleSortType = (value: SortType) => {
+    setSortType(value);
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        { activeTodosLength }
+        items left
```

Comment:
There is no space between the number and the text - use a string template instead, i.e 
```
`${count} text`
```

Reply:
also, active todos are todos with `compleated === false`